File: app/src/main/java/com/intelegain/agora/model/KnowdlegeAttachmentResponce.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by kalpesh.c on 1/3/18.
 */

public class KnowdlegeAttachmentResponce {


    @Expose
    @SerializedName("Attachments")
    public List<Attachments> Attachments;

    public static class Attachments {
        @Expose
        @SerializedName("FileData")
        public String FileData;
        @Expose
        @SerializedName("FileName")
        public String FileName;
    }
}

File: app/src/main/java/com/intelegain/agora/model/KnowdlegeAttachmentResponce.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by kalpesh.c on 1/3/18.
 */
class KnowdlegeAttachmentResponce {
    @kotlin.jvm.JvmField
    @Expose
    @SerializedName("Attachments")
    var Attachments: List<Attachment>? = null

    class Attachment {
        @Expose
        @SerializedName("FileData")
        var FileData: String? = null
        @kotlin.jvm.JvmField
        @Expose
        @SerializedName("FileName")
        var FileName: String? = null
    }
}